vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -236,107 +228,178 @@ An `env.json` file to configure the application URLs and client credentials must
 
 Instructions to export a bundle, including how to configure `env.json`, can be found in the [Export and Publish a Bundle tutorial](../../tutorials/create/pb/export-bundle-from-application.md).
 
-## Profile Management
-To manage and switch between different ent configurations, use `ent profile`. 
 
-To switch between different Entando Applications, even if they are in different clusters, `ent profile` uses Kubernetes contexts, kubeconfig files, and/or custom commands (refer to `ent profile first-use-readme`).
+### Custom Resource Wrapper
+Entando supports a series of `ent ecr` commands to facilitate custom resource operations. 
 
-It is common practice for the vendor tool that connects to a cloud Kubernetes to create a Kubernetes context that must be linked to the profile.
+#### Custom Resource Management
 
-An overview of the current connection and profile information is provided via `ent status`.
+Display the list of bundles associated with the current profile:
+```
+ent ecr list
+```
 
-1. Create and switch to a new profile
+Generate a custom resource and deploy it to the current profile:
 ```
-ent pro new [profileName] [EntandoAppName] [namespace]
+ent ecr deploy
+``` 
 
-This sets the minimal profile data and explains next steps.
+#### Determine Custom Resource Identifiers
```

Comment:
this confused me, Custom resource IDs?

Reply:
yes. per the text right beneath the header.

Reply:
Let's discuss this language a bit. Custom resource is a very general term and the ecr commands here are specifically and only for bundles. When you use ent ecr deploy, ent ecr deploy, etc. you are only dealing with a bundle which is single kind of custom resource, the EntandoDeBundle.